packages/babel-types/src/converters.js

```diff
@@ -1,7 +1,5 @@
 import isPlainObject from "lodash/isPlainObject";
-import isNumber from "lodash/isNumber";
 import isRegExp from "lodash/isRegExp";
```

Comment:
`isRegExp` doesn't have a `typeof` equivalent - those are the ones I'm targeting here.